test/kustomize/pvc.patch.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: internalstorage-postgres
+  namespace: clusterpedia-system
+spec:
+  select: null
```

Comment:
`selector`

Reply:
selector is not working here. its giving the issue mentioned on the slack